MEMBER_TRAVEL_FUND.md

```diff
@@ -1,137 +1,101 @@
 ## Purpose
 
-To establish and administer a fund for members of the Node.js
-Foundation to travel and spread knowledge about and support the foundation
-and the Node.js Foundation projects.
+To establish and administer a fund for members of the Node.js Foundation to travel and spread knowledge about and support Node.js ecosystem and the Foundation. 
 
-### [Current Years Fund][]
+### [Current Years Fund](https://github.com/nodejs/admin/blob/master/TravelFunds/2018.md)
 
 ## Restrictions
 
-* Candidates must be an individual member of the Node.js Foundation. The membership
-  status can be checked on [the foundation website][].
-* Requester must indicate that they do not have funding available from another source,
-  such as an employer willing to cover the costs or the event covering costs
-  for presenters or others.
+* Candidates must be an individual member of the Node.js Foundation. [The membership status can be checked on the foundation website](https://identity.linuxfoundation.org/user/login?destination=user/me). Due to privacy, the Individual Members list is not public. This team will verify the requester is on the Individual Members list before funds are disbursed. 
+* Those requesting funds must indicate that they do not have funding available from another source, such as an employer or the event itself that might covering costs for presenters. 
 
 ## Process
 
-Requests must be made prior to the start of travel.
-Requests submitted after travel has started will not be approved.
+Requests must be made prior to the start of travel. Requests submitted once travel is complete will not be approved. 
 
-Review of requests will take at least 72 hours and may take considerably longer.
-Approval is not guaranteed. If a request is rejected, reimbursement will not be
-issued. It is strongly recommended that you submit your request far enough in
-advance to get approval before making relevant expenditures.
+It is strongly recommended that you submit your request far enough in advance to get approval before making relevant expenditures. Review of requests will take at least 72 hours and may take considerably longer. Note, with limited funds available, approval is not guaranteed. If a request is rejected, reimbursement will not be issued.The Foundation cannot pre-pay for travel expenditures. 
```

Comment:
Are these under 80 chars? generally we try to keep our docs formatted to be 80 chars or less when possible.

Reply:
@mhdawson having a hard time understanding which docs should be under 80 chars?

Reply:
@mhdawson when you get a chance lmk about this comment!

Reply:
For our general markdown documentation we keep the length of each line of text to 80 chars or less.  For the second line that is part of the same paragraph you indent so that it lines up with the text from the first line.  I don't know if it matters in this case but it is what we follow in the core repo.